src/jobTree/jobStores/awsJobStore.py

```diff
@@ -76,7 +76,9 @@ def __init__( self, region, namePrefix, config=None ):
         self.jobs = self._getOrCreateDomain( 'jobs', create )
         self.versions = self._getOrCreateDomain( 'versions', create )
         self.files = self._getOrCreateBucket( 'files', create, versioning=True )
-        self.stats = self._getOrCreateBucket( 'stats', create )
+        self.stats = self._getOrCreateBucket( 'stats', create, versioning=True )
+        setattr(self, 'files', self.files)
```

Comment:
Nope, not needed.

`x.foo = 1` is equivalent to `setattr(self,'foo',1)`

They do exactly the same thing. Likewise for `self.foo` and `getattr(self,'foo')`.